megamek/src/megamek/client/ui/swing/unitDisplay/SummaryPanel.java

```diff
@@ -0,0 +1,71 @@
+package megamek.client.ui.swing.unitDisplay;
+
+import megamek.client.ui.swing.boardview.BoardView;
+import megamek.client.ui.swing.tooltip.PilotToolTip;
+import megamek.client.ui.swing.tooltip.UnitToolTip;
+import megamek.common.Entity;
+import megamek.common.EntityVisibilityUtils;
+import megamek.common.Hex;
+import megamek.common.Player;
+
+import javax.swing.*;
+
+public class SummaryPanel extends JPanel {
```

Comment:
This file needs a standard GPL header, and javadoc comments for the class and public constructors/methods.